source/simulator/parameters.cc

```diff
@@ -546,6 +546,13 @@ namespace aspect
                          "\n\n"
                          "Note that while more than one operation can be selected it only makes sense to "
                          "pick one rotational and one translational operation.");
+      prm.declare_entry ("Enable additional Stokes RHS", "false",
+                         Patterns::Bool (),
+                         "Whether to ask the material model for additional terms for the right-hand side "
+                         "of the Stokes equation. This can be used to implement force vectors for "
+                         "manufactured solution problems. The assembler for the Stokes system will "
+                         "create an additional output of type AdditionalMaterialOutputsStokesRHS.");
```

Comment:
the last sentence seems like a very detailed description of internals. there are many levels of knowledge to reach for a user to understand what this means.